api/tasks.go

```diff
@@ -883,6 +884,9 @@ func (a *TaskArtifact) Canonicalize() {
 	if len(a.GetterHeaders) == 0 {
 		a.GetterHeaders = nil
 	}
+	if a.Chown == nil {
+		a.Chown = pointerOf(false)
+	}
```

Comment:
If we're going to canonicalize it to the zero-value (`false`) anyways, we can make it `bool` with the tag `hcl:"chown,optional"` instead of a `*bool` and avoid downstream nil checking.

Reply:
Good idea! Fixed.